templates/main.go

```diff
@@ -94,3 +100,73 @@ func heartbeat() {
 		}
 	}
 }
+
```

Comment:
can we move this logic to `internal/file/file.go` ? 
so something like this https://github.com/commitdev/zero-deployable-backend/commit/bbc106d0c63a50c1c065774c24aea8669cbe0e21#diff-3201216eb4718ef228527e9e2f73c1b76363d1c1da3c86fc4d65065422ea6495

then we can have the routes to only get templated when fileUpload is enabled https://github.com/commitdev/zero-deployable-node-backend/blob/main/templates/src/app.js#L16-L17

Reply:
@larrylin17 ? This is still pending.